Bluetooth: mesh: samples: add support for nRF52840 dongle (#13979)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/light_dimmer/README.rst

After provisioning and configuring the mesh models supported by the sample in the `nRF Mesh mobile app`_, **Button 1** on the Mesh Light Dimmer device can be used to control the configured network nodes' LEDs, while **Button 2** can be used to store and restore scenes on the network nodes.

.. note::

>   When running this sample on a Thingy:53 or Dongle, the scene selection functionality will not be available as the device only has one button.
>   The single button of Thingy:53 and Dongle will have the dimming and on/off functionality as described for **Button 1** in this documentation.

Reviewer: ```suggestion
  When running this sample on Thingy:53 or the :ref:`zephyr:nrf52840dongle_nrf52840`, the scene selection functionality will not be available as the device only has one button.
  The single button of Thingy:53 and the dongle will be used for dimming and the on/off functionality as described for **Button 1** in this documentation.
```

   On long press and release, **Button 2** will publish a Scene Store message using the configured publication parameters of its model instance, and store the current LED state of all the targets under the scene with the most recently recalled scene number.

.. note::

>    :ref:`zephyr:thingy53_nrf5340` and :ref:`zephyr:nrf52840dongle_nrf52840` only supports the dimmer functionality, not scene selection.

Reviewer: ```suggestion
   :ref:`zephyr:thingy53_nrf5340` and the :ref:`zephyr:nrf52840dongle_nrf52840` only support dimming and the on/off functionality, not the scene selection functionality.
```
